index.html

```diff
@@ -200,5 +202,18 @@ <h2 class="main-text">About Myself</h2>
             </ul>
         </div>
     </section>
+    <div class="footer-section">
+        <section class="contact-section">
+            <h1 class="contact-header">Contact me</h1>
+            <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <form id="contact-form" action="https://formspree.io/f/xayzpebg" method="POST">
+                <input class="name-box" type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
+                <input class="email-box" type="email" name="email" placeholder="loremipsum@gmail.com" maxlength="500" required>
+                <textarea class="message-box" rows="8" cols="42" name="message" placeholder="Write your message here..." required></textarea>
```

Comment:
- [ ] Please from the project requirement, the message field should accept up to 500 chars. So make sure that you give the `maxlength` attribute to the message field. :pray: